modules/umo-editor.yml

```
@@ -0,0 +1,17 @@
+name: umo-editor
+description: Quickly integrate Umo editor into your Nuxt4 applications
+repo: umodoc/editor-nuxt
+npm: '@umoteam/umo-editor-nuxt'
+icon: umo-editor.svg
+github: https://github.com/umodoc/editor-nuxt
+website: https://github.com/umodoc/editor-nuxt
+website: https://www.umodoc.com
```

YAMLlint (1.38.0)

[error] duplication of key "website" in mapping

(key-duplicates)